tests/DependencyInjection/ContaoManagerExtensionTest.php

```diff
@@ -52,11 +53,29 @@ public function testInstantiation()
     }
 
     /**
-     * Tests the prepend() method.
+     * Tests the prepend() method does nothing if no plugin loader is there.
      */
-    public function testPrepend()
+    public function testPrependDoesNothingWhenNoPluginLoader()
     {
-        // TODO: add assertions
+        $container = $this->getMock(ContainerBuilder::class);
+        $container->expects($this->never())
```

Comment:
This would be the correct formatting according to the Sf code style:

```php
        $container = $this->getMock(ContainerBuilder::class);

        $container
            ->expects($this->never())
            ->method('get')
        ;
```

😄